lib/auth/apiserver.go

```diff
@@ -621,6 +621,9 @@ type upsertCertAuthorityRawReq struct {
 	TTL time.Duration   `json:"ttl"`
 }
 
+// upsertCertAuthority creates or updates a cert authority.
+// Deprecated: Use teleport.v1.trust.TrustService/UpsertCertAuthority instead.
```

Comment:
I don't think this deprecation notice does anything, deprecation doesn't apply to the same package and this is not exported (we're not failing any lint right now).